base/osutils.jl

```diff
@@ -13,13 +13,13 @@ such as a `ccall` to a non-existent function.
 """
 macro static(ex)
     if isa(ex, Expr)
-        if ex.head === :if || ex.head === :&& || ex.head === :||
+        if ex.head === :if || ex.head === :&& || ex.head === :|| || ex.head === :?
```

Comment:
It could, but that would be more of a new feature, since it was never able to before.